src/utils/git.go

```diff
@@ -30,13 +30,13 @@ func DoesTagExist(tag string) (bool, error) {
 }
 
 func OpenRepo() (*git.Repository, error) {
-	return git.PlainOpen(".")
+	return git.PlainOpenWithOptions(".", &git.PlainOpenOptions{DetectDotGit: true})
```

Comment:
This change makes OpenRepo open the repository within which the current directory is.
I.e. if we have a repo of the following structure:
```
repo
├── sub-dir-a
└── sub-dir-b
```

Running the command in `repo/sub-dir-a` will also succeed after this change. 
I don't think this should cause any problems